.pre-commit-config.yaml

```diff
@@ -11,6 +11,7 @@ repos:
       - id: detect-private-key
         exclude: (mock_credentials|_test\.go)
       - id: end-of-file-fixer
+        exclude: (\.txt)
```

Comment:
This pre-commit hook was adding extra whitespace to the mock_credentials files that contain fake usernames and passwords and was breaking them